### ISSUE TITLE
Breaks .NET Core 2.0 Assembly in macOS

### ISSUE THREAD

**Issue by desmortum** [lextm/obfuscar#135]
So I have a "Hello, World!" project as .NET Core 2.0 console app created in Visual Studio for Mac (former Xamarin Studio) and it's built as **test.dll**. It can be ran using **dotnet**.

Since Obfuscar itself seems to be a Mono assembly I do "mono obfuscar.console.exe obfuscar.xml" and it works producing new **test.dll**.

But it's not running with **dotnet** anymore. It crashes. Instead, it can be ran using **mono** now just fine. So somehow it's being recompiles as Mono assembly.

Any Ideas how to run Obfuscar on .NET Core assembly in macOS?

**Comment by watsonsong** [comment]
I met the same problem. I add a target in csproj before publich:
`
<Target Name="Obfuscate" BeforeTargets="Publish">
    <Copy SourceFiles="$(PublishDir)MyProj.dll" DestinationFolder="$(PublishDir)Original" />
    <Copy SourceFiles="$(PublishDir)MyProj.Console.dll" DestinationFolder="$(PublishDir)Original" />
    <Exec Command="$(Obfuscar) Obfuscar.xml" />
  </Target>
`

I can not execute the $(Obfuscar) on macos, I add the mono before it can work fine on macos.

**Comment by lextm** [comment]
There is a .NET Core Global Tool test build,

https://blog.lextudio.com/make-obfuscar-a-global-tool-in-net-core-1cc3ee2cfe30 

.NET Core SDK 2.1 and above is required.

**Comment by watsonsong** [comment]
Yes, but I still like the DotNetCliToolReference way, because the dependency environment is write into the project and no need all developers setup their environment on their own machine.

**Comment by jianyunli** [comment]
I am a newbie to Mac, how do you get the obfuscar installed on Mac?
